egs2/TEMPLATE/enh1/scripts/utils/calculate_speech_metrics.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+# 2021 @wangyou-zhang
+# Copied from ./scripts/utils/perturb_data_dir_speed.sh
+# Modified for calculating speech related metrics
+
+# Copyright 2021  Shanghai Jiao Tong University (author: Wangyou Zhang)
+# Apache 2.0
+
+# This script calculates the specified metric between audios in <ref_scp> and <enh_scp>,
+# and store it in the specified output file.
+
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+min() {
+  local a b
+  a=$1
+  for b in "$@"; do
+      if [ "${b}" -le "${a}" ]; then
+          a="${b}"
+      fi
+  done
+  echo "${a}"
+}
+
+export LC_ALL=C
+
+ref_channel=0         # reference channel
+nj=32       # The number of parallel jobs in inference
+python=python3        # Specify python to execute espnet commands
+id_prefix="enh-"      # prefix of utt ids to remove in <ref_scp> and <enh_scp>
+
+. utils/parse_options.sh
+. ./path.sh
+. ./cmd.sh
+
+
+if [[ $# != 4 ]]; then
+    echo 'Usage: calculate_speech_metrics.sh <ref_scp> <enh_scp> <metric> <outfile>'
```

Comment:
It is better to use help_message, which accept "--help".
https://github.com/espnet/espnet/blob/674bb25190a753920592195b419fd56e2fca2c85/egs2/TEMPLATE/asr1/asr.sh#L130

Reply:
Done.